src/Credentials/Credentials.php

```diff
@@ -101,6 +110,7 @@ public function __unserialize($data)
         $this->secret = $data['secret'];
         $this->token = $data['token'];
         $this->expires = $data['expires'];
+        $this->accountId = $data['accountId'];
```

Comment:
This change broke my application in production. The `Credentials` class is serialized by `CredentialProvider::cache()`. Previously cached credentials don't have this key, resulting in an `Undefined array key "accountId"` error exception.

Reply:
Hi @devfrey, apologies for the disruption.  Definitely an oversight on our part.  https://github.com/aws/aws-sdk-php/pull/2996 will most likely go out in today's release.

Reply:
Thank you for the swift resolution!